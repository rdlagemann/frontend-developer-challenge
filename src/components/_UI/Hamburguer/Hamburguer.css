.Hamburguer {
  border: none;
  background: transparent;
  position: fixed;
  top: 5px;
  right: 5px;
  z-index: 2;
}

.Hamburguer:hover,
.Hamburguer:focus {
  outline: none;
}

.Hamburguer__icon {
  padding: 15px;
  width: 50px;
}
