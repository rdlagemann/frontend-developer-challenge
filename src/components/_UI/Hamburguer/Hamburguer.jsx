import React from 'react'
import menu from './assets/menu.svg'
import cancel from './assets/cancel.svg'
import './Hamburguer.css'

export default function Hamburguer({ isOpen, onClick }) {
  return (
    <button className="Hamburguer" onClick={onClick}>
      {!isOpen && <img className="Hamburguer__icon" src={menu} alt="Menu" />}
      {isOpen && (
        <img className="Hamburguer__icon" src={cancel} alt="Close menu" />
      )}
    </button>
  )
}
