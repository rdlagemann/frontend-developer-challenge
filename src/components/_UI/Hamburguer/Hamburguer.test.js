import React from 'react'
import { create } from 'react-test-renderer'
import Hamburguer from './Hamburguer'

describe('Hamburguer', () => {
  it('render', () => {
    const output = create(<Hamburguer />)
    expect(output).toMatchSnapshot()
  })
})
