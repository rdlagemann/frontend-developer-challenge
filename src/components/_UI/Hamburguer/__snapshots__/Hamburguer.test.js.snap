// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburguer render 1`] = `
<button
  className="Hamburguer"
>
  <img
    alt="Menu"
    className="Hamburguer__icon"
    src="menu.svg"
  />
</button>
`;
